shared/fs/banner/conflict-banner-container.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import * as Types from '../../constants/types/fs'
+import {namedConnect} from '../../util/container'
+import ConflictBanner, {getHeight} from './conflict-banner'
+import * as RowTypes from '../row/types'
+import openUrl from '../../util/open-url'
+
+type OwnProps = {|path: Types.Path, conflictState: Types.ConflictState|}
```

Comment:
`conflictState` is available from `_tlf`, so it seems we should just have the banner accept a `path`.

Reply:
It was done like this so that `asRows` would always have the right height. But since it's not going to scroll, we can clean it up :)